setup.py

```diff
@@ -64,12 +64,8 @@ def get_version():
     extras_require={
         "docs": ["sphinx", "sphinx_rtd_theme", "recommonmark", "sphinxcontrib-bibtex", "sphinx_markdown_tables", "nbsphinx", "IPython"],
         "tests": ["pytest", "cirq", "ply", "sklearn"],
-        # Backends dependencies
-        "qibotf": ["qibotf"],
-        "qibojit": ["qibojit"],
-        "tensorflow": [f"tensorflow>=2.2.0"],
     },
-    python_requires=">=3.6.0",
+    python_requires=">=3.7.0",
```

Comment:
I guess with this change it will not be possible to install qibo at all in py3.6. In principle we could relax this given that for the most part, excluding some tensorflow and scipy stuff which are not hard requirements, qibo still works fine on py3.6. I am not sure if it is a good practice to support a version that we don't test and we don't include in the docs.

Reply:
I agree, but given that py36 is EOD, if we don't have a strong motivation (= technical need), we can simply skip this requirement and instead, include py3.10 as soon as it is supported by our backend dependencies.